### DISCUSSION TITLE
抽到了奇怪的体系，求配队

### DISCUSSION THREAD

**Discussion by xuanke66** [share121/inter-knot#3089] [其他]
常驻猫又 狼叔引擎 丽娜，怎么配队啊?
目前1队鲨鱼 安比 苍角，2队猫又 露西 派派
（2队犹豫中。。。。。。）

**Comment by xuanke66** [comment]
.<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >

**Comment by yitan1741** [comment]
等狼叔就好啦，丽娜暂时还是有点尴尬的，<img src="https://bbs-static.miyoushe.com/static/2024/08/07/61a221ea5beaa59b4c124f8c121489c8_336410838321628737.png" alt="艾梅莉埃-记录" width="75" >
